projects/plugins/backup/src/js/components/Backups.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import StatBlock from './StatBlock';
+import restApi from '../tools/jetpack-rest-api-client';
+import './backups-style.scss';
+import PostsIcon from './posts.svg';
+import CloudIcon from './cloud.svg';
+import UploadsIcon from './uploads.svg';
+import PluginsIcon from './plugins.svg';
+import ThemesIcon from './themes.svg';
+
+class Backups extends React.Component {
+	state = {
+		progress: 0,
+		latest: '',
+		stats: {
+			posts: 0,
+			uploads: 0,
+			plugins: 0,
+			themes: 0,
+		},
+		capabilities: [],
+		loaded: {
+			capabilities: false,
+			stats: false,
+		},
+	};
+
+	componentDidMount() {
+		restApi.setApiRoot( this.props.apiRoot );
+		restApi.setApiNonce( this.props.apiNonce );
+		restApi.fetchRecentBackups().then( res => {
+			// If we have no backups don't load up stats
+			if ( res.length === 0 ) {
+				return;
+			}
+
+			const latestBackup = res[ 0 ];
+			if ( latestBackup.stats ) {
+				this.setState( {
+					stats: {
+						plugins: latestBackup.stats.plugins.count,
+						themes: latestBackup.stats.themes.count,
+						uploads: latestBackup.stats.uploads.count,
+						posts: latestBackup.stats.tables.wp_posts.post_published, // TODO: Update endpoint to give back posts information instead of relying on wp_posts
+					},
+					loaded: {
+						stats: true,
+						capabilities: this.state.loaded.capabilities,
+					},
+				} );
+			}
+
+			if ( latestBackup.progress ) {
+				this.setState( { progress: latestBackup.progress } );
+			} else {
+				this.setState( { progress: 0 } );
+			}
+
+			if ( latestBackup.status === 'finished' && latestBackup.last_updated ) {
+				this.setState( { latest: latestBackup.last_updated } );
+			}
+		} );
+
+		// Mocked with simulated delay - Will pull data from end point
+		// Simulates a modified /rewind/capabilities response
+		setTimeout( () => {
+			this.setState( {
+				capabilities: [ 'backup', 'realtime-backup' ],
+				loaded: {
+					capabilities: true,
+					stats: this.state.loaded.stats,
+				},
+			} );
+		}, 1000 );
+	}
+
+	render() {
+		return (
+			<div className="jp-wrap">
+				<div className="jp-row">
+					<div className="lg-col-span-3 md-col-span-4 sm-col-span-4">
+						<div className="backup__latest">
+							<img src={ CloudIcon } alt="Jetpack Logo" />
+							<h2>Latest Backup</h2>
+						</div>
+						<h1>{ moment.utc( this.state.latest ).fromNow() }</h1>
```

Comment:
Maybe we could rely on `@wordpress/date` and its `dateI18n` to format those kinds of date displays?

Reply:
Maybe, I'm not sure it supports verbage close to the design.  If it does, definitely.  Currently it's displaying things like "A few seconds ago" "2 hours ago" through the use of moment's `fromNow()`.  I'll take a look at what we're doing on cloud.jetpack.com to see if there are some more i18n friendly things going on.

Reply:
Updated to use i18n date functionality